packages/frontend/core/src/modules/navigation/view/use-register-navigation-commands.ts

```diff
@@ -43,6 +44,22 @@ export function useRegisterNavigationCommands() {
       })
     );
 
+    unsubs.push(
+      registerAffineCommand({
+        id: 'alert-ctrl-s',
+        category: 'affine:general',
+        preconditionStrategy: PreconditionStrategy.Never,
+        keyBinding: {
+          binding: '$mod+s',
+        },
+        label: '',
+        icon: null,
+        run() {
+          toast('save');
```

Comment:
Please add i18n for `save`.

Reply:
i am resolve all the things please check #6548
@pengx17